### PR TITLE
Fix DCS issues

### PR DESCRIPTION
**Pull request by outcatcher**
## Summary of the Pull Request
Change DCS schema
Add missing schema attributes
Fix `max_memory` type in the schema
Remove `ForceNew` from `backup_policy`

## PR Checklist

* [x] Refers to: #639
* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed
```
=== RUN   TestAccDcsInstancesV1_basic
--- PASS: TestAccDcsInstancesV1_basic (736.50s)
PASS

Process finished with exit code 0

=== RUN   TestAccDcsInstancesV1_depr
--- PASS: TestAccDcsInstancesV1_depr (389.97s)
PASS

Process finished with exit code 0
```

### CODE REVIEW

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1.go [hunk 1]
```diff
@@ -93,30 +98,75 @@ func resourceDcsInstanceV1() *schema.Resource {
 				Computed: true,
 			},
 			"save_days": {
-				Type:     schema.TypeInt,
-				Required: true,
-				ForceNew: true,
+				Type:       schema.TypeInt,
+				Optional:   true,
+				Deprecated: "Please use `backup_policy` instead",
```
Comment (lego963) [inline opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1.go]: Add `ConflictsWith:` here and beyond deprecated fields

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go [hunk 2]
```diff
@@ -84,36 +130,106 @@ func testAccCheckDcsV1InstanceExists(n string, instance instances.Instance) reso
...
+  port = "8002"
+  code = "%s"
+}
+data "opentelekomcloud_dcs_product_v1" "product_1" {
+  spec_code = "dcs.master_standby"
+}
+resource "opentelekomcloud_dcs_instance_v1" "instance_1" {
+  name              = "%s"
+  engine_version    = "3.0.7"
+  password          = "Huawei_test"
```
Comment (lego963) [inline opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go]: change password

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go [hunk 3]
```diff
@@ -84,36 +130,106 @@ func testAccCheckDcsV1InstanceExists(n string, instance instances.Instance) reso
...
```
Comment (lego963) [inline opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go]: same

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go [hunk 4]
```diff
@@ -84,36 +130,106 @@ func testAccCheckDcsV1InstanceExists(n string, instance instances.Instance) reso
...
+  port = "8002"
+  code = "%s"
+}
+data "opentelekomcloud_dcs_product_v1" "product_1" {
+  spec_code = "dcs.single_node"
+}
+resource "opentelekomcloud_dcs_instance_v1" "instance_1" {
+  name              = "%s"
+  engine_version    = "3.0.7"
+  password          = "Huawei_test"
```
Comment (lego963) [inline opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1_test.go]: same

File: opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1.go [hunk 5]
```diff
@@ -77,6 +79,10 @@ func resourceDcsInstanceV1() *schema.Resource {
 				ForceNew: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+			"created_at": {
```
Comment (lego963) [inline opentelekomcloud/resource_opentelekomcloud_dcs_instance_v1.go:82]: add this param to doc please

**Review by lego963 — DISMISSED** [review]

**Comment by outcatcher** [comment]
recheck

**Review by lego963 — APPROVED** [review]

**Pull request merged**